Not suitable for upstream since upstream has changed this with 3.26

Fix for compiling and linking gsm-resources.c on Solaris

--- a/src/Makefile.am	2017-11-20 13:45:05.160631723 -0800
+++ b/src/Makefile.am	2017-11-20 13:45:50.853005567 -0800
@@ -42,11 +42,11 @@
 gnome_system_monitor_SOURCES = \
 	$(gnome_system_monitor_h_files) \
 	$(gnome_system_monitor_cpp_files) \
+	gsm-resources.c \
 	main.cpp
 
 # See legacy/Makefile.am for explanation
 LDADD = \
-	legacy/gsm-resources.o \
 	legacy/libgsmlegacy.la \
 	$(GSM_LIBS) \
 	$(SYSTEMD_LIBS) \
